process/track/baseBlockTrack.go

```diff
@@ -697,9 +697,10 @@ func (bbt *baseBlockTrack) doWhitelistIfNeeded(metablock *block.MetaBlock) {
 	miniblocks := metablock.GetMiniBlockHeaders()
 	keys := make([][]byte, 0)
 	for _, miniblock := range miniblocks {
-		receiverInShard := miniblock.GetReceiverShardID() == shardID
-		senderCross := miniblock.GetSenderShardID() != shardID
-		if receiverInShard && senderCross {
+		senderShard := miniblock.GetReceiverShardID()
```

Comment:
I think there is an error here (at least in the naming of the variable)

Reply:
done